docs/modules/deployment/pages/time-series-storage/timeseries/cortex.adoc

```diff
@@ -0,0 +1,79 @@
+The plugin uses the xref:deployment:time-series-storage/timeseries/ts-integration-layer.adoc[time series storage integration layer] to convert metrics to a Prometheus model and delegate writes and reads to https://cortexmetrics.io/[Cortex].
+The plugin also makes it possible for users to visualize graphs from the Cortex instance in https://docs.opennms.com/helm/latest/index.html[OpenNMS Helm], through the OpenNMS performance data source or the https://grafana.com/grafana/plugins/prometheus/[Prometheus Data Source].
+
+.Cortex Time Series Plugin
+image::time-series-storage/cortex-plugin.png[Cortex plugin architecture,500]
+
+== Requirements
+
+Make sure you have the following before you start to use the Cortex plugin:
+
+* Horizon 30.0.0+ (Meridian availability with the 2023 release)
+* Cortex version 1.10.0+
+
+NOTE: We plan to keep the plugin compatible with https://grafana.com/oss/mimir/[Mimir], Grafana's fork of Cortex, as long as Mimir remains compatible with Cortex.
+The minimum supported Mimir version is 2.0.0.
+
+== Deployment
+
+To use the Cortex Time Series Plugin, do the following:
+
+. Start Cortex.
++
+See https://cortexmetrics.io/docs/getting-started/.
++
+If your aim is to get up to speed quickly in a test environment, you can use Docker Compose to deploy a https://github.com/opennms-forge/stack-play/tree/master/standalone-cortex-minimal[minimal, standalone version of Cortex].
+
+. Download the plugin's `.kar` file from GitHub (https://github.com/OpenNMS/opennms-cortex-tss-plugin/releases).
+. Copy the file into the `opennms/deploy` directory of your running instance.
+. Enable and configure the time series storage.
++
+[source, console]
+----
+echo "org.opennms.timeseries.strategy=integration
+org.opennms.timeseries.tin.metatags.tag.node=${node:label}
+org.opennms.timeseries.tin.metatags.tag.location=${node:location}
+org.opennms.timeseries.tin.metatags.tag.geohash=${node:geohash}
+org.opennms.timeseries.tin.metatags.tag.ifDescr=${interface:if-description}
+org.opennms.timeseries.tin.metatags.tag.label=${resource:label}" >> ${OPENNMS_HOME}/etc/opennms.properties.d/cortex.properties
+----
+
+. Configure the plugin.
++
+Note that you can omit this step if you use the default settings.
++
+[source, console]
+----
+config:edit org.opennms.plugins.tss.cortex
+
+property-set writeUrl http://localhost:9009/api/prom/push
+property-set readUrl http://localhost:9009/prometheus/api/v1
+property-set maxConcurrentHttpConnections 100
+property-set writeTimeoutInMs 1000
+property-set readTimeoutInMs 1000
+property-set metricCacheSize 1000
+property-set externalTagsCacheSize 1000
+property-set bulkheadMaxWaitDurationInMs 9223372036854775807
+
+config:update
+----
+. Configure the plugin to remain installed after reboot:
+
+[source,console]
+----
+echo 'opennms-plugins-cortex-tss wait-for-kar=opennms-cortex-plugin' > "$OPENNMS_HOME/etc/featuresBoot.d/cortex.boot"
+----
+
+== Cortex tips
```

Comment:
For consistency with my changes above, please also change `cortex-service-ip-addr` below to `cortex-service-host`.